frontend/public/components/storage-class-form.tsx

```diff
@@ -55,7 +57,105 @@ export class StorageClassForm_ extends React.Component<
+    parameters: {},
+  };
+
+  CSIStorageTypes = {
+    'rbd.csi.ceph.com': {
+      title: 'Ceph RBD',
+      provisioner: 'rbd.csi.ceph.com',
+      documentationLink: 'https://rook.github.io/docs/rook/master/ceph-block.html',
+      parameters: {
+        clusterID: {
+          name: 'Cluster ID',
+          hintText: 'Cluster ID is the namespace where Ceph is deployed.',
+          required: true,
+        },
+        pool: {
+          name: 'Pool',
+          hintText: 'Ceph pool into which the RBD image shall be created',
+          required: true,
+        },
+        imageFormat: {
+          name: 'Image Format',
+          hintText: 'Ceph RBD image format.',
+          required: true,
+        },
+        imageFeatures: {
+          name: 'Image Features',
+          hintText: 'Ceph RBD image features.',
+          required: true,
+        },
+        'csi.storage.k8s.io/provisioner-secret-name': {
+          name: 'Provisioner Secret Name',
+          hintText: 'Provisioner Secret Name is the name of provisioner secret.',
+          required: true,
+        },
+        'csi.storage.k8s.io/provisioner-secret-namespace': {
+          name: 'Provisioner Secret Namespace',
+          hintText: 'Provisioner Secret Namespace is the namespace where Ceph is deployed.',
+          required: true,
+        },
+        'csi.storage.k8s.io/node-stage-secret-name': {
+          name: 'Node Stage Secret Name',
+          hintText: 'Node Stage Secret Name is the name of Node Stage secret.',
+          required: true,
+        },
+        'csi.storage.k8s.io/node-stage-secret-namespace': {
+          name: 'Node Stage Secret Namespace',
+          hintText: 'Node Stage Secret Namespace is the namespace where Ceph is deployed.',
+          required: true,
+        },
+        'csi.storage.k8s.io/fstype': {
+          name: 'Filesystem Type',
+          hintText: 'Ceph RBD filesystem type. Default set to ext4.',
+          required: true,
+        },
+      },
+    },
+    'cephfs.csi.ceph.com': {
+      title: 'Ceph FS',
+      provisioner: 'cephfs.csi.ceph.com',
+      documentationLink: 'https://rook.github.io/docs/rook/master/ceph-filesystem.html',
+      parameters: {
+        clusterID: {
+          name: 'Cluster ID',
+          hintText: 'Cluster ID is the namespace where Ceph is deployed.',
+          required: true,
+        },
+        fsName: {
+          name: 'Filesystem Name',
+          hintText: 'CephFS filesystem name into which the volume shall be created',
+          required: true,
+        },
+        'csi.storage.k8s.io/provisioner-secret-name': {
+          name: 'Provisioner Secret Name',
+          hintText: 'Provisioner Secret Name is the name of provisioner secret.',
+          required: true,
+        },
+        'csi.storage.k8s.io/provisioner-secret-namespace': {
+          name: 'Provisioner Secret Namespace',
+          hintText: 'Provisioner Secret Namespace is the namespace where Ceph is deployed.',
```

Comment:
Provisioner Secret Namespace is the namespace where provisioner secret is created, this need to updated for other hintText also